website/docs/examples/multi_client_geoimage/README.md

```diff
@@ -0,0 +1,236 @@
+---
+id: multi-client-geoimage
+title: Multi Client GeoImage
+---
+
```

Comment:
We created a standard-ish message for the top of these examples about running locally vs ssh'd into the robot. Suggest to copy and combine the details at the top of the GPS client and the Camera client examples, and tweak to be corect